distrib/sets/lists/minix-base/md.i386

```diff
@@ -60,6 +60,8 @@
 ./usr/lib/libm387.so.0                                  minix-base
 ./usr/lib/libm387.so.0.1                                minix-base
 ./usr/lib/libm387_pic.a                                 minix-base
+./usr/lib/libacpi.a  	                                minix-base
```

Comment:
This is really nitpicking but the columns are supposed to line up. I can't see here if this is due to the use of a tab character, but the distrib files use spaces only.